data/json/monsters/mutant_mammal.json

```diff
@@ -364,5 +445,160 @@
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "400000 ml",
+    "weight": "400 kg",
+    "hp": 270,
+    "speed": 145,
+    "attack_cost": 110,
+    "material": [ "flesh" ],
+    "symbol": "D",
+    "color": "light_gray",
+    "morale": 30,
+    "aggro_character": false,
+    "melee_skill": 6,
+    "melee_dice": 4,
+    "melee_dice_sides": 3,
+    "melee_damage": [ { "damage_type": "cut", "amount": 8 } ],
+    "armor_elec": 1,
+    "dodge": 2,
+    "armor_bash": 2,
+    "anger_triggers": [ "HURT", "PLAYER_NEAR_BABY", "FRIEND_ATTACKED", "PLAYER_CLOSE" ],
+    "harvest": "mutant_mammal_large_leather",
+    "dissect": "dissect_lupine_sample_single",
+    "families": [ "prof_intro_biology" ],
+    "reproduction": { "baby_monster": "mon_bigdog_greatestswiss_pup", "baby_count": 4, "baby_timer": 360 },
+    "//2": "Given the way bigger pups, decreased their number and increased gestation time",
+    "biosignature": { "biosig_item": "feces_dog", "biosig_timer": 2 },
+    "zombify_into": "mon_dog_zombie_brute",
+    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PET_MOUNTABLE", "CANPLAY", "PATH_AVOID_DANGER_1", "WARM" ]
+  },
+  {
+    "id": "mon_bigdog_greatestswiss_pup",
+    "type": "MONSTER",
+    "name": { "str": "Greatest Swiss puppy", "str_pl": "Greatest Swiss puppies" },
+    "description": "An adorable, defenseless Greatest Swiss puppy.  Much safer to tame than an adult dog, in spite of its already impressive size.",
+    "default_faction": "dog",
+    "bodytype": "dog",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "7500 ml",
+    "weight": "10 kg",
+    "hp": 24,
+    "speed": 105,
+    "material": [ "flesh" ],
+    "symbol": "d",
+    "color": "light_gray",
+    "aggression": -11,
+    "morale": 4,
+    "melee_skill": 2,
+    "melee_dice": 1,
+    "melee_dice_sides": 1,
+    "melee_damage": [ { "damage_type": "cut", "amount": 3 } ],
+    "dodge": 1,
+    "vision_night": 5,
+    "harvest": "dog_small_with_skull",
+    "path_settings": { "max_dist": 10 },
+    "upgrades": { "age_grow": 42, "into": "mon_bigdog_greatestswiss" },
+    "zombify_into": "mon_dog_zombie_rot",
+    "biosignature": { "biosig_item": "feces_dog", "biosig_timer": 6 },
+    "fear_triggers": [ "FRIEND_ATTACKED", "FRIEND_DIED", "HURT" ],
+    "special_attacks": [ [ "EAT_FOOD", 100 ] ],
+    "petfood": {
+      "food": [ "DOGFOOD" ],
+      "feed": "The %s seems to like you!  It lets you pat its head and seems friendly.",
+      "pet": "The %s happily wags its tail while you pat their head."
+    },
+    "flags": [ "ANIMAL", "DOGFOOD", "CANPLAY", "HEARS", "KEENNOSE", "PATH_AVOID_DANGER_1", "SEES", "SMELLS", "WARM" ]
+  },
+  {
+    "id": "mon_bigdog_gottweiler",
+    "type": "MONSTER",
+    "name": { "str": "Gottweiler" },
+    "description": "This Rottweiler has grown to the proportions of a cow.  Its short fur is riddled with scars and the hide underneath has grown thick and leathery.",
```

Comment:
Needs a bit more flavor. Size of a cow and scarred is not a bad start, but it's also not something that pops.